packages/venia-ui/lib/components/MegaMenu/submenuColumn.js

```diff
@@ -23,8 +23,8 @@ const SubmenuColumn = props => {
     let children = null;
 
     if (category.children.length) {
-        const childrenItems = category.children.map((category, index) => {
-            const { url_path, isActive, name } = category;
+        const childrenItems = category.children.map((subCategory, index) => {
```

Comment:
Isn't this a variable name change? How did this fix the issue?

Reply:
Line 25 checks for category.children.length, but then line 26 was changing the category variable. A couple lines later we would evaluate category.children.length, but it was undefined on the sub category. This fix is just undoing the variable overwritting that was happening